### DISCUSSION TITLE
Does Wings use TCP or UDP?

### DISCUSSION THREAD

**Discussion by OpenSource03** [pterodactyl/panel#3079] [General Community]
Hello!

I'd like to ask if Wings uses TCP or UDP?

Thanks!

**Comment by schrej** [comment]
The communication between Wings and the panel is using HTTP and therefore TCP (with the latest and greatest HTTP versions it could also be UDP, but wings doesn't use those). For the game servers, ports for both are allocated, what gets used depends on the game.

**Reply by TheRealGramdalf** [reply]
I'm having some issues with this - trying to run a server that runs geysermc but it just won't work, the java server runs and is accessible, but the bedrock one won't. I tried changing ports, trying the official geyser-floodgate-paper egg, using purpur egg with geysermc on top, but nothing works. Any clues on where to find the docker network configuration?

**Reply by docimin** [reply]
I think bedrock uses a different port than java. Are you sure you have at least 2 allocations - allocated to the server?
